app/Http/Controllers/TaskController.php

```diff
@@ -35,21 +28,20 @@ public function store(Request $req){
         }
         else{
 
-            $houre = date('H', strtotime($req->time));
-            if ($houre > 12) {
-                $houre = $houre - 12;
-                $time = date($houre .':i A', strtotime($req->time));
+            $date2 = Task::where('date', $req->date)->get();
+            if ($date2->count() > 0) {
+                return ['status' => 'error', 'msg' => 'Date Already Taken!'];
             }
             else{
-                $time = date('H:i A', strtotime($req->time));
-            }
+                $date6 = date('Y-m-d A', strtotime($req->date));
+                // return $date;
 
-            $data = new Task;
-            $data->name = $req->name;
-            $data->time = $time;
-            $data->date = $req->date;
-            if($data->save()){
-                return ['status' => 'success', 'msg' => 'Data has been Inserted.'];
```

Comment:
You need to return `@return \Illuminate\Http\Response` not array
https://laravel.com/docs/9.x/responses#response-objects